proto/google/events/cloud/build/v1/events.proto

```diff
@@ -0,0 +1,31 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package google.events.cloud.build.v1;
+
+option csharp_namespace = "Google.Events.Protobuf.Cloud.Build.V1";
+
+import "google/events/cloud/build/v1/data.proto";
+import "google/events/cloudevent.proto";
+
+// The CloudEvent raised when your build's state changes.
+message CloudBuildEvent {
+  option (google.events.cloud_event_type) =
+      "google.cloud.build.v1.event";
```

Comment:
Do we only have one event type? In general the type should look like something `google.cloud.build.something.v1.happened`

Reply:
Yep. Currently we only have one event type for cloudbuildsource